projects/core/src/qualtrics/config/default-qualtrics-config.ts

```diff
@@ -0,0 +1,8 @@
+import { QualtricsConfig } from './qualtrics-config';
+
+export const defaultQualtricsConfig: QualtricsConfig = {
+  qualtrics: {
+    active: false,
+    multi: false,
```

Comment:
This is a feature for SPA, we can't detect it because it must manually be done by the user.

For example, in the screenshot below, this must be done manually.

the flow would be like this, we will have a global qualtricscomponent at anytime that will be put in the footer. So at any point of a page, it will be evaluated once, but if they want it evaluated multiple times, they would put qualtricscomponent in another page with a slot that dies when leaving the page, so that it will call it again when they visit that page.

![Screen Shot 2019-10-07 at 11 29 47 AM](https://user-images.githubusercontent.com/29099466/66325842-f1a0f580-e8f5-11e9-8c1e-2f665f0ca8b3.png)